app/assets/javascripts/change_password.js

```diff
@@ -0,0 +1,16 @@
+jQuery(function($) {
+    $('#new_password_confirmation, #new_password, #password').keyup(function() {
+      var passwordPresent = $('#password').val().length > 0;
+      var button = $('input[name="commit"]');
+      var newPass = $('#new_password').val();
+      var newPassConfirm = $('#new_password_confirmation').val();
+      var newPasswordsMatches = newPass === newPassConfirm;
+      var newPasswordsPresent = (newPass.length > 0) && (newPassConfirm.length > 0);
+
+      if  (passwordPresent && newPasswordsMatches && newPasswordsPresent) {
+        button.removeAttr('disabled');
+      } else {
+        button.attr('disabled','disabled');
+      }
+    });
+});
```

Comment:
BTW, the button is enabled at first. So when no value has been entered, the form can be submitted, which is inconsistent.